jobs/config.json

```diff
@@ -6723,6 +6723,8 @@
       "--gcp-project=kubernetes-scale",
       "--gcp-zone=us-east1-a",
       "--ginkgo-parallel=30",
+      "--gke-command-group=beta",
+      "--gke-create-command=container clusters create --quiet --enable-ip-alias --create-subnetwork=ip-alias-subnet --cluster-ipv4-cidr=/12 --services-ipv4-cidr=/19",
```

Comment:
the /12 would allow only a max of 4K nodes. Is that the intent?

Reply:
Yes - this is a 2k-node test.